packages/odyssey-react/src/components/Tabs/Tabs.module.scss

```
@@ -12,26 +12,28 @@
 
 .tablist {
   position: relative;
-  border-block-end: 1px solid $border-color-display;
+  border-block-end-width: var(--BorderBlockEndWidthTabList);
+  border-block-end-style: var(--BorderBlockEndStyleTabList);
+  border-block-end-color: var(--BorderBlockEndColorTabList);
 }
 
 .tab {
   display: inline-block;
   position: relative;
   margin-block: 0;
   margin-inline: 0;
-  padding-block: $spacing-s;
-  padding-inline: $spacing-s;
+  padding-block: var(--SpacePaddingTab);
+  padding-inline: var(--SpacePaddingTab);
   border: 0;
   background: none;
-  color: $text-body;
+  color: var(--ColorTab);
```

```suggestion
  color: var(--ColorTextTab);
```

This is one of the rougher edges of our current naming system but we're already grouping global `ColorText` tokens this way so we should continue this approach for component variables here to keep the API consistent